components/atoms/carousel/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { Carousel } from 'react-responsive-carousel'
+import 'react-responsive-carousel/lib/styles/carousel.min.css'
+
+class Carrousel extends PureComponent {
```

Comment:
I think it's not necessary to use PureComponent

Reply:
`PureComponent` can improve our page performance, so we should use it when is possible.

Reply:
Also, I think this component should be stateless

Reply:
for sure @beatriz1304 !!!